### ISSUE TITLE
CPU_FTR_P9_TLBIE_BUG should be cleared on Nimbus 2.3 and Cumulus 1.3 or later

### ISSUE THREAD

**Issue by mpe** [linuxppc/issues#253]
This should be skiboot's job, via the device tree CPU features, but we just hacked it into the kernel:

```
	if ((version & 0xffff0000) == 0x004e0000) {
		cur_cpu_spec->cpu_features &= ~(CPU_FTR_DAWR);
		cur_cpu_spec->cpu_features |= CPU_FTR_P9_TLBIE_BUG;
		cur_cpu_spec->cpu_features |= CPU_FTR_P9_TIDR;
	}
```

**Comment by mpe** [comment]
This should be skiboot's job, via the device tree CPU features, but we just hacked it into the kernel:

```
	if ((version & 0xffff0000) == 0x004e0000) {
		cur_cpu_spec->cpu_features &= ~(CPU_FTR_DAWR);
		cur_cpu_spec->cpu_features |= CPU_FTR_P9_TLBIE_BUG;
		cur_cpu_spec->cpu_features |= CPU_FTR_P9_TIDR;
	}
```

**Comment by mpe** [comment]
Fixed in torvalds/linux@677733e296b5c7a37c47da391fc70a43dc40bd67.
